Felienne/hedy
# Add back-to-class button to the class stats page #1917

**Please fill out this template by replacing all content between < >**

< Add back button to the class stats page >

**Description**

< Changes in detail > Use present tense without a subject to describe your PR, for example: "Adds translations of levels 1 to 12 to Polish"

**Fixes < #1917  >**

< Always link the number of the issue or of the discussion that your pr concerns. >
Notable exception: adding content may be done without an accompanying issue 

**How to test**

- Run Hedy locally and log in as a teacher.
- Navigate to /for-teachers and pick a class. Then select "Show class statistics" to go to the /stats/class/$class-id page.
- Check that this page contains a "Back to class" button in the top right corner and that the button correctly navigates to the /for-teachers/class/$class-id page.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
